internal/translator/data/rules.go

```diff
@@ -34,6 +36,106 @@ type Rules struct {
 // RulesHelper has useful functions to work with the collection
 type RulesHelper interface {
 	FindByPattern(ctx context.Context, source, target, pattern string) ([]Rule, error)
+	NewRule(ctx context.Context, ruleStruct Rule) (*primitive.ObjectID, error)
```

Comment:
`NewRule` it's better as `New`, because  you are in context of `Rules` so if you execute something like `Rules.New` you already know that you are creating a rule
`Rules.NewRule` is redundant.

Reply:
I agree